### DISCUSSION TITLE
Official Obsidian Clipper

### DISCUSSION THREAD

**Discussion by Nitero** [DominikPieper/obsidian-ReadItLater#182] [General]
So now that the official [Obsidian Clipper](https://github.com/obsidianmd/obsidian-clipper) was released I fear that this plugin will become a bit redundant...
The extension fulfills a very similar purpose and has quite the extensive syntax to pull out data so you can create any source you need, solving issues like [this](https://github.com/DominikPieper/obsidian-ReadItLater/issues/142) and [this](https://github.com/DominikPieper/obsidian-ReadItLater/issues/150)

The only reason I see to still keep this plugin around is that it has API access to YouTube, meaning I can save a video without first having to open it in a new tab, but apart from that I'm not sure what makes it unique anymore... anyone else got specific use-cases they stay for?

**Comment by adamluckdev** [comment]
Maybe I'm as a maintainer of this plugin biased, but I think ReadItLater and the official Clipper can coexist. Clipper si right now better at creating notes from webpages, because it has access to complete rendered DOM in a browser. ReadItLater has custom parsers, it uses some APIs for richer content, settings are managed in the Obsidian app so they can be synced accros your devices and has support for running in mobile version of Obsidian.

I just want to point out that everything has its pros and cons and only users will decide what works better for them.

**Reply by Nitero** [reply]
oh I wasn't aware the clipper doesn't work on mobile, thats a big one!

**Reply by craziedde** [reply]
@adamluckdev,

### Yes, correct, I think ReadItLater is very helpful on the mobile

I'm trying to configure those properties in default ReaditLater templates, I think I will be able to just need to configure all variables correctly

<img width="850" height="719" alt="image" src="https://github.com/user-attachments/assets/e84a8a72-57c3-4ac3-a0af-5442dd971897" />




> **Would be amazing to create 'quite' option
To be able to share to ReaditLater on mobile/android without 1) opening Obsidian 2) pressing "ReaditLater**

<img width="967" height="1287" alt="image" src="https://github.com/user-attachments/assets/f601eb3a-9d7d-4d3a-85a9-57d5a83e1da1" />
